k4e-agent-sti.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
+
+set -e
+
+# The scripts performs the following steps on rhel-8.5:
+# * Install imagebuilder 
+# * Create rpms for yggdrasil and k4e-device-worker
+# * Publish rpms as yum repo to be consumed by imagebuilder
+# * Publish aarch64 rhel-image for edge-device
+
+# The script assumes the machine is registered via subscription-manager
+if ! subscription-manager refresh; then
+   echo  "The machine is not registered."
+   exit 1
+fi
+
+#---------------------------
+# Install osbuild components
+#---------------------------
+dnf install -y osbuild-composer composer-cli cockpit-composer bash-completion
+systemctl enable osbuild-composer.socket --now
+systemctl enable cockpit.socket --now
+systemctl start firewalld
+firewall-cmd --add-service=cockpit && firewall-cmd --add-service=cockpit --permanent
+source  /etc/bash_completion.d/composer-cli
+
+#-----------------------------------
+# Build packages for k4e from source
+#-----------------------------------
+mkdir /home/builder
+
+dnf install -y dbus-devel systemd-devel git golang rpm-build
+rm -rf ~/rpmbuild/rpmbuild/*
+
+# Build yggdrasil rpm
+git clone https://github.com/jakub-dzon/yggdrasil.git /home/builder/yggdrasil
+cd /home/builder/yggdrasil
+export CGO_ENABLED=0
+
+export ARCH=$(uname -i)
+GOPROXY=proxy.golang.org,direct PWD=$PWD spec=$PWD outdir=$PWD make -f .copr/Makefile srpm
```

Comment:
this is from the k4e-agent-build correct? can we use a make target/rule instead? I mean, put these lines in 'make XXX' and from both scripts call it? or call the k4e-agent-build script?

Reply:
I'd rather this script not to depend on any other component. so it can simply be invoked by 
```
curl -s -L https://raw.githubusercontent.com/masayag/r4e/k4e-sti/k4e-agent-sti.sh | bash
```

Reply:
if we do it this way then all scripts will have their copy of the code.
Someone might want to do: